<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="left" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>480</width>
        <height>480</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>480</width>
        <height>480</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <widget class="QLabel" name="camera">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>480</width>
         <height>480</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 1px solid #333;  /* 显示边界 */
qproperty-alignment: AlignCenter;  /* 居中裁剪 */</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="circle">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>60</y>
         <width>266</width>
         <height>266</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>266</width>
         <height>266</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>266</width>
         <height>266</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/images/circle.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QWidget" name="success" native="true">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>360</y>
         <width>280</width>
         <height>55</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>280</width>
         <height>55</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>280</width>
         <height>55</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(49, 50, 54, 63);
border-radius: 10px;</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>55</width>
            <height>55</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>55</width>
            <height>55</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">border-image: url(:/images/right.png);
background-color: rgba(255, 255, 255, 0);</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>5</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="minimumSize">
           <size>
            <width>110</width>
            <height>55</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>55</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 25 24pt &quot;Songti SC&quot;;
border: none;
background-color: rgba(49, 50, 54, 0);
color: rgb(255,255,255)</string>
          </property>
          <property name="text">
           <string>认证成功</string>
          </property>
          <property name="scaledContents">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="indent">
           <number>-1</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="right" native="true">
      <property name="minimumSize">
       <size>
        <width>320</width>
        <height>480</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>320</width>
        <height>480</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(41, 45, 54,0.9);</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="_title" native="true">
         <property name="maximumSize">
          <size>
           <width>320</width>
           <height>55</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="title">
            <property name="font">
             <font>
              <family>Songti SC</family>
              <pointsize>24</pointsize>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 24pt &quot;Songti SC&quot;;
color: rgb(255,255,255);</string>
            </property>
            <property name="text">
             <string>人脸识别智能考勤系统</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="_avatar" native="true">
         <property name="minimumSize">
          <size>
           <width>320</width>
           <height>160</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>320</width>
           <height>160</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="avatar">
            <property name="minimumSize">
             <size>
              <width>160</width>
              <height>160</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>160</width>
              <height>160</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">border-radius: 80px;
border: 1px solid #333;  /* 显示边界 */
qproperty-alignment: AlignCenter;  /* 居中裁剪 */</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="_info" native="true">
         <property name="minimumSize">
          <size>
           <width>320</width>
           <height>250</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>320</width>
           <height>250</height>
          </size>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QWidget" name="idInfo" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QLabel" name="label_6">
               <property name="styleSheet">
                <string notr="true">color: rgb(95, 110, 127);</string>
               </property>
               <property name="text">
                <string>工号</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="employeeIDEdit">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="styleSheet">
                <string notr="true">color: rgb(255, 255, 255);

border: 1px solid #313236;
border-radius: 4px;</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="nameInfo" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <item>
              <widget class="QLabel" name="label_8">
               <property name="styleSheet">
                <string notr="true">color: rgb(95, 110, 127);</string>
               </property>
               <property name="text">
                <string>姓名</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="nameEdit">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="styleSheet">
                <string notr="true">color: rgb(255, 255, 255);

border: 1px solid #313236;
border-radius: 4px;</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="partInfo" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_9">
             <item>
              <widget class="QLabel" name="label_10">
               <property name="styleSheet">
                <string notr="true">color: rgb(95, 110, 127);</string>
               </property>
               <property name="text">
                <string>部门</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="departmentEdit">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="styleSheet">
                <string notr="true">color: rgb(255, 255, 255);

border: 1px solid #313236;
border-radius: 4px;</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="dateInfo" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_11">
             <item>
              <widget class="QLabel" name="label_14">
               <property name="styleSheet">
                <string notr="true">color: rgb(95, 110, 127);</string>
               </property>
               <property name="text">
                <string>时间</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="dateEdit">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="styleSheet">
                <string notr="true">color: rgb(255, 255, 255);

border: 1px solid #313236;
border-radius: 4px;</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
